atomic_reactor/plugins/pre_resolve_remote_source.py

```diff
@@ -108,13 +109,23 @@ def run(self):
         }
 
     def set_worker_params(self, source_request, remote_source_url, remote_source_conf_url):
-        build_args = {
-            # Turn the environment variables into absolute paths that
-            # represent where the remote sources are copied to during
-            # the build process.
-            env_var: os.path.join(REMOTE_SOURCE_DIR, value)
-            for env_var, value in source_request.get('environment_variables', {}).items()
-        }
+        build_args = {}
+        # This matches values such as 'deps/gomod' but not 'true'
+        rel_path_regex = re.compile(r'^[^/]+/[^/]+(?:/[^/]+)*$')
```

Comment:
Should we keep a list of variables that should be converted to absolute paths or have Cachito to also return additional information (e.g., type) on the env variables? This way we wouldn't need to worry if additional variables are added in the future using a slash for some different reason or if we get a relative path with no nested entries (e.g., converting 'usr' to /usr).

Reply:
@athos-ribeiro the idea is that Cachito will eventually be enhanced to provide the environment variable type. We can discuss this on Monday to determine if I should do that first in Cachito and then patch OSBS the right way or pursue the approach in the PR as a quick fix.